man/vaeac_print_train_summary.Rd

```diff
@@ -4,7 +4,7 @@
 \alias{vaeac_print_train_summary}
 \title{Function to printout a training summary for the \code{vaeac} model}
 \usage{
-vaeac_print_train_summary(best_epoch, best_epoch_running, last_state)
+vaeac_print_train_summary(best_epoch, best_epoch_running, last_epoch)
```

Comment:
Rebuild manual after changing back to `last_state` in the function call and rather updating the oxygen @param.